modules/spo-uninstall-console.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="spo-uninstall-console_{context}"]
+= Uninstall the Security Profiles Operator using the web console
+
+To remove the Security Profiles Operator, you must first delete the seccomp profiles. After the profiles are removed, you can then remove the Operator and its namespace by deleting the *openshift-security-profiles* project.
+
+.Prerequisites
+
+* Access to an {product-title} cluster that uses an account with `cluster-admin` permissions.
+* The Security Profiles Operator is installed.
+
+.Procedure
+
+To remove the Security Profiles Operator by using the {product-title} web console:
+
+. Navigate to the *Operators* -> *Installed Operators* page.
+
+. Delete all `seccomp` profiles.
```

Comment:
Before "Uninstall Operator" on GUI, below resource should also be deleted. Otherwise recreate operator will be impacted. 
`mutatingwebhookconfiguration` spo-mutating-webhook-configuration